src/scss/components/_colleges-grid.scss

```diff
@@ -0,0 +1,163 @@
+.college-blocks {
+  overflow: auto;
+  background: #ffcc00;
+}
+
+.college-block {
+  width: 50%;
+  float: left;
+  overflow: hidden;
+  position: relative;
+
+  @media (min-width: $screen-sm-min) {
+    width: calc(100%/3);
+  }
+
+  @media (min-width: $screen-md-min) {
+    width: calc(100%/4);
+  }
+
+  @media (min-width: $screen-lg-min) {
+    width: calc(100%/5);
+  }
+
+  .college-block-img {
+    width: 100%;
+    transition: -webkit-filter .3s;
+    transition: filter .3s;
+    transition: filter .3s, -webkit-filter .3s;
+    z-index: 1;
+
+    -webkit-filter: grayscale(1) sepia(.15) contrast(.65) brightness(.65);
+    filter: grayscale(1) sepia(.15) contrast(.65) brightness(.65);
+  }
+
+  .college-block-text {
+    font-family: "Knockout 49 A","Knockout 49 B",Georgia,serif;
```

Comment:
Also on .college-block-text, it looks like you need to add a z-index of 2 to fix a display issue in firefox.  Oddly it only happens on the homepage and not the college pages...